src/Scaffold/Console/controller/_list_toolbar.stub

```diff
@@ -2,20 +2,20 @@
     <a
         href="<?= Backend::url('{{lower_author}}/{{lower_plugin}}/{{lower_name}}/create') ?>"
         class="btn btn-primary oc-icon-plus">
-        New {{title_singular_name}}
+        <?= e(trans('backend::lang.form.create_title', ['name' => '{{title_singular_name}}'])) ?>
```

Comment:
Translations have to be added via another flag, the default has to be untranslated for simplicity.

Reply:
@LukeTowers what do you mean by "via another flag" ?

Reply:
I mean unless `--include-translations` is passed to the command they should remain plain text. See https://github.com/octobercms/october/issues/2833#issuecomment-299303061 for why

Reply:
Oh, I see what you mean now.

Reply:
The other PR introduced a specific language file for the plugin, this uses the backend translations which are already in place and are used everywhere else in the core. I dont see why we would add this as a flag, as it would make the stub files extremely complex for a marginal benefit (It looks a bit less scary to beginners)

Reply:
@alxy it doesn't add that much complexity, only a simple conditional around each block of text. Not including the translation strings in these stub files was a conscious decision that @daftspunk made from the start, and it actually does help beginners a lot. Everything that a beginner encounters is an abstraction that they have to figure out before they can move on, so minimizing that by not including the strings in default generated files helps them get started. Later on they learn about translation strings and how useful they are, and then they can start using the `--include-translations` flag like everyone else.